packages/react/src/components/Dialog/Dialog.base.tsx

```diff
@@ -174,14 +171,9 @@ export class DialogBase extends React.Component<IDialogProps, {}> {
       >
         <DialogContent
           subTextId={this._defaultSubTextId}
-          title={dialogContentProps.title}
-          subText={dialogContentProps.subText}
           showCloseButton={mergedModalProps.isBlocking}
-          topButtonsProps={dialogContentProps.topButtonsProps}
-          type={dialogContentProps.type}
-          onDismiss={onDismiss ? onDismiss : dialogContentProps.onDismiss}
-          className={dialogContentProps.className}
           {...dialogContentProps}
+          onDismiss={onDismiss ? onDismiss : dialogContentProps.onDismiss}
```

Comment:
With the previous setup, `dialogContentProps.onDismiss` would always override `onDismiss`. If that behavior is desirable to preserve (I'm not sure offhand), this line should just be `onDismiss={onDismiss}` and move back up above `...dialogContentProps`.

Reply:
I moved it down to respect the order of the explicit statement, but you're right, I'll play it safe and keep it as it was before.